File: benchmark/comparison/results/blind_ts/16.ts
function contains(lst: number[], target: number): boolean {
  for (const x of lst) {
    if (x === target) return true;
  }
  return false;
}

console.log(String(contains([1, 2, 3], 2)));
console.log(String(contains([1, 2, 3], 4)));
console.log(String(contains([], 1)));

File: benchmark/comparison/results/blind_py/16.py
def contains(lst, target):
    for x in lst:
        if x == target:
            return True
    return False

def fmt(b):
    return "true" if b else "false"

print(fmt(contains([1, 2, 3], 2)))
print(fmt(contains([1, 2, 3], 4)))
print(fmt(contains([], 1)))
